src/calliope_pathways/math/pathways.yaml

```diff
@@ -317,31 +317,43 @@ global_expressions:
     foreach: [nodes, techs, carriers, costs, investsteps]
     equations:
       - where: flow_cap_new
-        expression: sum($cost_sum * flow_cap_new, over=vintagesteps)
+        expression: sum($cost_sum * flow_cap_new * available_vintages, over=vintagesteps)
```

Comment:
What is `$cost_sum` referring/doing here? I was not able to find it anywhere else...

Reply:
It is defined in https://github.com/calliope-project/calliope/blob/872978dfe3a305c85e0c279925fb794d970bf6bd/src/calliope/math/base.yaml#L796